src/main/scala/chisel3/util/circt/LTLIntrinsics.scala

```diff
@@ -77,6 +89,30 @@ private[chisel3] object LTLDelayIntrinsic {
   }
 }
 
+/** A wrapper intrinsic for the CIRCT `ltl.repeat` operation. */
+private[chisel3] object LTLRepeatIntrinsic {
+
+  def apply(base: Int, more: Option[Int])(_in: Bool)(implicit sourceInfo: SourceInfo) = {
+    val params = more match {
+      case None    => Seq("base" -> IntParam(base))
+      case Some(l) => Seq("base" -> IntParam(base), "more" -> IntParam(l))
+    }
```

Comment:
```suggestion
    val params = Seq("base" -> IntParam(base)) ++ more.map("more" -> IntParam(_))
    }
```
No need to repeat the `base` bit.

Reply:
changed it for delay as well (I had written that part too)